src/content/1.7/development/form-types/category-choice-tree.md

```diff
@@ -0,0 +1,53 @@
+---
+title: CategoryChoiceTreeType
+menuTitle: CategoryChoiceTreeType
+weight: 2
+---
+
+### CategoryChoiceTreeType
+
+CategoryChoiceTreeType is a child of [MaterialChoiceTreeType](material-choice-tree), it extends parent options
+with options listed bellow. It is used to display category tree selection box and requires 
```

Comment:
> It is used to display category tree selection box

IMO a screenshot of the rendered form type would make it easier for developers to understand what they can expect from each form type.